CONTRIBUTING.md

```diff
@@ -174,34 +174,15 @@ Please copy and paste the following quotes for the languages:
 
 #### Environment Variable For Apple M1 Platform
 
-To set the Docker platform environment variable in your terminal tab, run:
+To set the following environment variable seems not be needed anymore, probably because Docker knows in its newest version on which CPU it runs and sets that to default.
+But we leave this command here to keep the knowledge about how to set the platform:
```

Comment:
If you encounter trouble building the docker containers on an Apple M1 chip you can try to explicitly define the target platform docker builds and pulls images for:

Reply:
Cool text!
Done